app/assets/stylesheets/tweets.css.scss

```diff
@@ -0,0 +1,10 @@
+.tweet {
+  height: 200px;
```

Comment:
I don't like such CSS. maybe min-height needs to be added.

OR jquery-match-height JS added (better) because on different screen resolution it might not work

Reply:
try to check with different resolutions. how it works